web/themes/new_weather_theme/templates/block/block--weathergov-current-conditions.html.twig

```diff
@@ -1,104 +1,85 @@
-{# these first three should go away when we put this into a region #}
-<div class="grid-container weather-gov-current-conditions padding-0 margin-0">
-  <div class="grid-row">
-    <div class="grid-col-6">
-{#=======================================#}
-
-      <div class="grid-container padding-0 margin-0">
-        {# First row: place name and timestamp #}
-        <div class="grid-row">
-          <div class="grid-col-12">
-            <h1>{{ content["#data"]["location"] }}</h1>
-            <p class="margin-0 font-body-3xs text-gray-50">
-              {{ 'Weather as of' | t }}
-              {# Datetimes are localized in client-side Javascript. #}
-              <weather-timestamp data-utc="{{ content["#data"]["timestamp"]["utc"] }}">
-                {{ content["#data"]["timestamp"]["formatted"] }}
-              </weather-timestamp>
-            </p>
-          </div>
-        </div>
-
-        {# Second row: widget-type interface #}
-        <div class="grid-row">
```

Comment:
As noted in the comment, this all goes away because there's a region now that handles setting it up.